packages/plugin-doc-coverage/src/lib/config.unit.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+
+describe('docCoveragePluginConfigSchema', () => {
+  it('accepts a documentation coverage configuration with all entities', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        language: 'typescript',
+        sourceGlob: 'src/**/*.{ts,tsx}',
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
+
+  it('accepts minimal configuration with only language', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        language: 'javascript',
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
```

Comment:
```suggestion
 it.each([
    'javascript',
    'typescript'
  ])('accepts configuration with only the language property set to %s', (language) => {
    expect(() =>
      docCoveragePluginConfigSchema.parse({
        language,
      } satisfies DocCoveragePluginConfig),
    ).not.toThrow();
  });
```

You could use a each block the iterate over all different languages. 
To also not double test the functionality in the test `accepts both typescript and javascript as valid languages` I renamed the test a bit.